src/tensors/cpu/intgemm_interface.h

```diff
@@ -258,6 +314,25 @@ class PrepareBiasForBNodeOp : public NaryNodeOp {
     //std::cerr << "TrueBias: " << child(0)->name() << " type: " << child(0)->type() << " bQuantMult: " << this->child(3)->val()->data()[0] <<  " aQuantMult: " << this->child(2)->val()->data()[0] << std::endl;
     //std::cerr << "Bias name and val: " << child(0)->name() << " " << child(0)->val()->data()[0] << std::endl;
 #ifdef COMPILE_CPU
+  #if defined(WASM)
+    return {NodeOp(
+      if (alreadyPrepared_) {
+        //God Knows why trying to assign the bias tensor to this node causes a crash, the second time it's referenced
```

Comment:
What's going on here?

Reply:
Is there any reason this code was copied instead of a more focused ifdef around the specific call?